id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/utils/IdentifiableTest.scala

```diff
@@ -16,4 +16,15 @@ class IdentifiableTest extends FunSpec with PropertyChecks with Matchers {
     }
   }
 
+  // We have a rule that identifiers can't start with a number, as this
+  // makes them awkward to use as a string in certain contexts.
+  it("should never generate an identifier that starts with a number") {
```

Comment:
Not sure what everyone thinks, but I think we should have one comment either in the test or in the actual code explaining why we do this. Not in both places as it is harder to ensure we keep things up to date. I personally prefer it being in tests as one of the purposes of tests should be documentation